routes/web.php

```diff
@@ -1,8 +1,22 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\FAQController;
 
 Route::get('/', function () {
     return view('welcome');
 })->name('home');
 
+Route::get('profile', function () {
+    return view('profile');
+})->name('profile');
+
+Route::get('dashboard', function () {
+    return view('dashboard');
+})->name('dashboard');
+
+/*
+ * The routes about tasks
+ */
+Route::get('/faq', [FAQController::class, 'index'])->name('faq.index');
```

Comment:
This does not add routes for the controller methods you added for the create scenario. But I also see that there was no create view implemented as well.